src/main/kotlin/team/themoment/gsmNetworking/global/security/jwt/TokenParser.kt

```diff
@@ -1,22 +1,42 @@
 package team.themoment.gsmNetworking.global.security.jwt
 
+import io.jsonwebtoken.Claims
+import io.jsonwebtoken.Jwts
 import org.springframework.http.HttpStatus
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken
 import org.springframework.stereotype.Component
 import team.themoment.gsmNetworking.common.exception.ExpectedException
 import team.themoment.gsmNetworking.global.security.jwt.properties.JwtProperties
+import team.themoment.gsmNetworking.global.security.principal.AuthenticationDetailsService
+import java.security.Key
+import javax.servlet.http.HttpServletRequest
 
 /**
  * 토큰을 파싱하는 클래스 입니다.
  */
 @Component
-class TokenParser {
+class TokenParser(
+    private val authenticationDetailsService: AuthenticationDetailsService,
+    private val jwtProperties: JwtProperties
+) {
 
     /**
-     * 재발급 토큰의 prefix를 파싱하는 역할의 메서드 입니다.
+     * access 토큰의 prefix를 파싱하는 메서드 입니다.
+     *
+     * @param request 요청한 requestServlet
+     * @return token prefix를 제외한 access token
+     */
+    fun parseAccessToken(request: HttpServletRequest): String? =
+        request.getHeader(JwtProperties.HEADER)
+            .let { it ?: return null }
+            .let { if (it.startsWith(JwtProperties.TOKEN_PREFIX)) it.replace(JwtProperties.TOKEN_PREFIX, "") else null }
```

Comment:
이 코드에서 scope함수를 사용하는 이유를 모르겠습니다. 
let함수는 엘비스 연산자와 함께 사용해 null검사를 간결하게 하거나, 변수의 scope를 제한해 더 코드를 간결하게 읽히기 위해 사용하는데 장점을 잘 모르갰습니다 저는 아래와 같이 풀어쓰는 것을 권장합니다.

```suggestion
       val jwtHeaderValue = request.getHeader(JwtProperties.HEADER)
           ?: return null
        return if (jwtHeaderValue.startsWith(JwtProperties.TOKEN_PREFIX)) { 
                    jwtHeaderValue.replace(JwtProperties.TOKEN_PREFIX, "") 
               } else { null }
        }
```

Reply:
scope 함수를 사용한 이유는 그저 메서드를 풀어 쓴거 보다 더 간결해 보여서 사용하였던거 같습니다.
코멘트 해주신대로 scope 함수를 사용했을때 얻는 장점을 고려하지 작성하지 않았던거 같습니다.

scope 함수를 사용하지 않되, 코멘트 해주신 `jwtHeaderValue` 변수명 보단 `accessToken`을 파싱하는 메서드이니 `accessToken` 변수명을 사용하는 식으로 수정하도록 하겠습니다.